rbi/core/array.rbi

```diff
@@ -1847,6 +1847,8 @@ class Array < Object
     )
     .returns(T.any(T.nilable(Elem), T::Array[Elem]))
   end
+  sig { returns(Elem) }
+  sig { params(random: Random::Formatter).returns(Elem) }
```

Comment:
we don't support keyword arguments in overloading.
You get either overloading or keyword args.

Reply:
Aha -- should I close this out then?  It doesn't seem like there is anything beyond either dropping the keyword args (which doesn't seem acceptable as it constrains the type system to not allow a perfectly valid call) vs having a type that is inexact but still technically correct (i.e. not overloading).